mmdet3d/datasets/pipelines/loading.py

```diff
@@ -197,7 +199,8 @@ def __call__(self, results):
                     cloud arrays.
         """
         points = results['points']
-        points.tensor[:, 4] = 0
+        if points.tensor.shape[-1] >= 5:
+            points.tensor[:, 4] = 0
```

Comment:
This hack is error-prone, please modify or add new options in the init to achieve this. Please also note to fix the unit test.